vendor/engines/c2po/app/services/credit_card_account_builder.rb

```diff
@@ -0,0 +1,20 @@
+# Contains overrides for building a `CreditCardAccount` from params.
+# Dynamically called via the `AccountBuilder.for()` factory.
+class CreditCardAccountBuilder < AccountBuilder
+
+  # Override and append `build`.
+  def build
```

Comment:
Hmmm, sure.  Do you think calling `super` a code smell for some reason or are you looking for enhanced clarity?